includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -363,6 +363,28 @@ public function refund_charge( $charge_id, $amount = null ) {
 		return $this->request( $request, self::REFUNDS_API, self::POST );
 	}
 
+	/**
+	 * List refunds
+	 *
+	 * @param string $charge_id - The charge to retrieve the list of refunds for.
+	 * @param string $payment_intent_id - The payment intent to retrieve the list of refunds for.
+	 *
+	 * @return array
+	 * @throws API_Exception - Exception thrown on request failure.
+	 */
+	public function list_refunds( string $charge_id = null, string $payment_intent_id = null ) {
+		$query = [ 'limit' => 100 ];
```

Comment:
Minor: I suspect this doesn't matter since we're asking for a specific charge_id, but do you think we can set this limit to 1?

Reply:
We shouldn't limit it to one, since Stripe will return all the refunds for the charge, including failures, or previous partial refunds, which may not match the refund we're looking for.

You've made me realise that there is a potential for a charge to have lots of refunds on it and then become impossible to refund here, but I think that's very unlikely for an interac_present refund.